test/conformance/kuttl/verifyImages/clusterpolicy/standard/configmap-context-lookup/README.md

```diff
@@ -0,0 +1,11 @@
+## Description
+
+The configmap context lookup uses informer's cache internally, the background processing should use the same to resolve configmap context without crashing Kyverno.
+
+## Expected Behavior
+
+Policy creation should not cause Kyverno to panic.
```

Comment:
Similar note as with the other test. Policy is expected to be successfully created AND not result in an internal panic.